### ISSUE TITLE
How to honor the zenith point (top elevation) 

### ISSUE THREAD

**Issue by kaki222** [cgre-aachen/gempy#538]
Dear GemPy Forum
I have almost fitted the surface using a good number of data but only, the surface stops at the upper most contour without closing. 
How can I close the surface by honoring the top elevation point (X_top, Y_top, Z_top)=Last point in CONTOUR-40.

Also looking a the origin of my data (SPE paper). It is an isopach.
is it fair to think of just a flat basement?
I attach the code (Adaptation of Dr. Miguel) and data (webdigitizer: extracted four contours, East-West Line and North-South Line and the Wells Positions). Any advise and code improvement are welcome.

By the way I am very new to GemPy. (Coming from ISDX Creo, formerly ProEngineer).

Thank you in advance. 

[Code.docx](https://github.com/cgre-aachen/gempy/files/5432808/Code.docx)
![Forum-Q2-PIC](https://user-images.githubusercontent.com/34009148/97080658-1570cf80-160e-11eb-83e6-1c8e75580f0b.jpg)
[data.zip](https://github.com/cgre-aachen/gempy/files/5432809/data.zip)

**Comment by AlexanderJuestel** [comment]
Hey @kaki222,

the reason your surface is not closing is probably because your vertical model extent is too small. Try to increase it in the direction of your top value :) 

Also, your example is a fantastic case to try out GemGIS (GemPy Geographic) as GemGIS is capable of exactly transforming your given data into usable data for GemPy -> https://github.com/cgre-aachen/gemgis. Let me know if you have any questions regarding GemGIS

**Comment by kaki222** [comment]
Yes increasing the Z extent did the trick. It all the time GemPy was calculating the surface. it was there but not shown.

I will try gemgis. 
I am now trying to most of the Docs-tutorials. But still can I ask you how to get orientations not necessarily at each data point but in few regions. As of now I have one major orientation (0,0,1).
Thanks Alexander.

**Comment by AlexanderJuestel** [comment]
Great!

I am ironically working on extracting orientations based on the distance of isolines right now. In the image below, you have isolines and I digitized the gradient at different positions (red arrows). Based on these arrows and a provided height difference, an orientation for GemPy can be calculated automatically! This feature will be available in a few days when we release the next minor version of GemGIS. So stay tuned! If you desperately need it, I can also provide a sneak preview for your case. 


![image](https://user-images.githubusercontent.com/45469915/97083863-6f27c880-1613-11eb-8354-aac02f827429.png)

**Comment by kaki222** [comment]
Thank you.
I can wait.

 but a lot of ideas in the attached for my isopach (equal thicknesses). however, I don't think I need that orientation accuracy as my model will be used to model flow of oil/gas which itself (reservoir simulation) will be part of a big pipeline optimization.

I have so many questions but I should be patient and do my homeworks.
With respect.

**Comment by AlexanderJuestel** [comment]
Hey, I will move this issue to GemGIS as it is more related to the data preparation than the actual modeling process ;)
